tests/kuttl/tests/manila_scale/01-assert.yaml

```diff
@@ -0,0 +1 @@
+../../common/assert_sample_deployment.yaml
```

Comment:
This has to be a symlink (it looks the same in github after comit though)
Example: https://github.com/openstack-k8s-operators/cinder-operator/pull/119/files#diff-914634bb88a93bef3c030b81fdd8c691a91f20c1478c9acecf4879fa2daf692c - 01-assert.yaml has the symlink folder sign